azext_iot/product/command_requirement.py

```diff
@@ -0,0 +1,13 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_iot.product.shared import BadgeType
+from azext_iot.product.providers.provider import get_sdk
+
+
```

Comment:
nit: for the other vNext implemented services, the command module is plural `commands_requirement` reason being a collection of commands against the resource (in this case resource being requirement).